docs/fundamentals/code-analysis/quality-rules/ca1865-ca1867.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "CA1865: Use char overload"
+description: "Use 'string.Method(char)' instead of 'string.Method(string)' when you have a string with a single char"
+ms.date: 07/27/2023
+ms.topic: reference
+f1_keywords:
+  - CA1865
+helpviewer_keywords:
+  - CA1865
+dev_langs:
+  - CSharp
+  - VB
+author: mrahhal
+---
+
+# CA1865-CA1867: Use 'string.Method(char)' instead of 'string.Method(string)' when you have a string with a single char
```

Comment:
```suggestion
# CA1865-CA1867: Use 'string.Method(char)' instead of 'string.Method(string)' for string with single char
```

Reply:
Not sure about this one as I actually had a similar message in the impl PR but it was corrected to [this](https://github.com/dotnet/roslyn-analyzers/pull/6799/files#diff-5d2487f8d1310f0861cdc9af913a873a8559b3a669bccc14dd1e44edca7d652dR2112). Should I still apply this suggestion?

Reply:
I guess you are talking about [this suggestion](https://github.com/dotnet/roslyn-analyzers/pull/6799#discussion_r1274237604). "When you have a..." seems too long-winded, especially for a title. I think "for" is better.